charts/icm-as/templates/jobserver-deployment.yaml

```diff
@@ -18,6 +18,14 @@ apiVersion: batch.core.intershop.de/v1
 kind: ICMJob
 metadata:
   name: {{ include "icm-as.fullname" $jobValues }}-job
+  {{- include "icm-as.podData" $jobValues | nindent 2 }}
```

Comment:
(about the whole section, not only the line) I would expect, that the section is part of the template-spec and not of the ICMJob object

Reply:
I know, currently the ICMJob-Definition may not contain metadata.annotations and metadata.labels for the job-template. This would be the better place.